src/App/App.js

```diff
@@ -9,17 +9,20 @@ function App() {
   const [movies, setMovies] = useState([]);
   const [selectedMovie, setSelectedMovie] = useState({});
   const [isMovieSelected, setIsMovieSelected] = useState(false);
+  const [isError, setIsError] = useState(false)
+  const [responseLevel, setResponseLevel] = useState(undefined)
```

Comment:
Typically we use `null` to set a "nothing" value as default. `undefined` isn't something we purposely set a value to.